io/src/test/scala/fs2/io/file/FileSpec.scala

```diff
@@ -127,6 +130,59 @@ class FileSpec extends BaseFileSpec {
     }
   }
 
+  "permissions" - {
+    "should fail for a non existent file" in {
+      Blocker[IO]
+        .use(b => file.permissions[IO](b, Paths.get("nothing")))
+        .attempt
+        .unsafeRunSync()
+        .isLeft shouldBe true
+    }
+    "should return permissions for existing file" in {
+      val permissions = PosixFilePermissions.fromString("rwxrwxr-x")
+      Blocker[IO]
+        .use { b =>
+          tempFile
+            .evalMap { p =>
+              IO.delay(Files.setPosixFilePermissions(p, permissions)) >> file.permissions[IO](b, p)
```

Comment:
I think it's fine to use the fs2 method for setting permissions - that way we'll verify that the methods interact well with each other. I'd wait for someone else to state their opinion on this, though.

Reply:
Yeah, I think that should be fine as well. It felt a bit odd when I first did it that way but will just be the same in the end and a bit tidier as well. I'll wait for someone to confirm before pushing a change.

Reply:
Yeah I think it's fine to use fs2 method here

Reply:
Updated to just use the fs2 methods in both tests